core/src/App.html

```diff
@@ -1203,6 +1204,12 @@
           tpcErrorHandling(thirdPartyCookiesCheck);
         }
       }
+
+      if ('storage' === e.data.msg) {
+        let getValidMessageSource = IframeHelpers.getValidMessageSource(e);
+        let microfrontendId = getValidMessageSource.luigi.id;
+        StorageHelper.process(microfrontendId, window.location.hostname, e.data.data.id, e.data.data.operation, e.data.data.params);
```

Comment:
Shouldn't it be 'e.origin' instead of 'window.location.hostname'?

Reply:
Let's do an example, we have:
1)Luigi core hosted in http:://luigi.core
2)Micro A hosted on http:://frontend.com/front1
3)Micro B hosted on http:://frontend.com/front2
4)Micro  C hosted on http:://another.com/front

We have two possibility:
#1 Using as key e.origin: 
in this case, MicroA and MicroB can share storage items.... MicroC (different domain) will not be able to share storage with A,B

#2 Using as key window.location.hostname: 
In this case all the Micros (A,B,C) can share it .....

I was thinking on option 2, that's y I was using window.location.hostname.....
@hardl If you think it is better scenario 1 (Only Micro fronted hosted in same domain can share items....) i'll just change it... I guess you know better than me the requirements... 

With solution 2, On other hand, .... with Remove and Clear you can delete items for all the Micro and create some problems......

Reply:
Option 1 is exactly, how it should behave ;)
This feature's intention is to act as an alternative to native local storage where it can't be used (third party cookie, private tab,...)

Reply:
Done, now we will use e.origin to build the storage key